### PR TITLE
Add validator frontend

### LINKED ISSUE #219: Display validator and Training Jobs
#### Software and Hardware Versions 
ModelCI v1.x.x, CUDA Version vx.x.x, GPU device used...

#### Problem description

- [ ] Display a one-minute validator, enable the user to have a quick view of validation accuracy
- [ ] Display a Training job table

#### Other Information 
Things you tried, stack traces, related issues, suggestions on how to fix it...

### PR DESCRIPTION
**Pull request by univerone**
Closes #219 

#### What has been done to verify that this works as intended?
Display validation progress
Add upload dataset button
Screenshot_2021-02-09 ML ModelCI(4)

Use the dropdown button to select operations on models
Screenshot_2021-02-07 ML ModelCI(7)

Display training jobs
Screenshot_2021-02-07 ML ModelCI(8)

#### Before submitting this PR, please make sure you have:

- [x] run `python -m pytest tests/` and confirmed all checks still pass.
- [x] verified that any code or assets from external sources are properly credited.

### CODE REVIEW

File: modelci/app/experimental/endpoints/cv_tuner.py [hunk 1]
```diff
@@ -131,7 +143,7 @@ def update_finetune_model_as_new(id: str, updated_layer: Structure, dry_run: boo
             framework=model.framework,
             engine=Engine.NONE,
             model_status=[ModelStatus.DRAFT],
-            version=ModelVersion(model.version.ver + 1),
+            version=version,
```
Comment (YuanmingLeee) [inline modelci/app/experimental/endpoints/cv_tuner.py:146]: Note that you may reconsider the primary key constraint on the model.

Comment (YuanmingLeee) [reply modelci/app/experimental/endpoints/cv_tuner.py:146]: the model has a primary key on (name, framework, engine, version). This won't allow the user to have multiple model draft version.

**Review by YuanmingLeee — APPROVED** [review]

**Comment by YuanmingLeee** [comment]
image
The validation accuracy including the value is too large. You can change to the same font size as the button below.
validation accuracy: -

**Comment by HuaizhengZhang** [comment]
also, validate -> quick validate

better organize "submit" "validate" and "train" vertically and design a more clean layout. Briefly introduce the button usage

the button of `import customized dataset` also should be placed

File: modelci/app/experimental/endpoints/cv_tuner.py (hunk 1 above)
Comment (univerone) [reply modelci/app/experimental/endpoints/cv_tuner.py:146]: Yes, these lines of code are trying to avoid model saving error due to duplicate draft version

**Comment by YuanmingLeee** [comment]
1.

Select:
- CIFAR10
- MNIST
- ImageNet
- ...
- Customized

When the user selects the customized dataset, the dataset uploader apear.

2. When the model is fast validation, the start training button is disabled
3. Validation accuracy no need bold

**Review by YuanmingLeee — APPROVED** [review]

**Pull request merged**